src/Microsoft.AspNet.Mvc.Core/ModelBinding/Metadata/DataAnnotationsMetadataProvider.cs

```diff
@@ -84,7 +84,7 @@ public void GetDisplayMetadata([NotNull] DisplayMetadataProviderContext context)
             // DisplayName
             if (displayAttribute != null)
             {
-                displayMetadata.DisplayName = displayAttribute.GetName();
+                displayMetadata.DisplayName = (propertyName) => displayAttribute.GetName();
```

Comment:
minor (and unrelated to perf :smiley_cat:) but don't we use `_` for unused lambda arguments?